packages/loading-indicator/src/LoadingIndicator.stories.tsx

```diff
@@ -74,5 +74,5 @@ const Template: StoryFn<typeof Spinner> = (
 
 export const LiveExample = Template.bind({});
 LiveExample.args = {
-  description: 'Loading...',
+  description: 'Loading dot dot...',
```

Comment:
non-blocking but curious what's the reason for changing the descriptions in the stories?

Reply:
i think to make it easier to tell that it's centered